client/src/components/Meetup/Meetup.js

```diff
@@ -1,8 +1,689 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
+import {
+  Avatar,
+  Box,
+  Button,
+  ButtonGroup,
+  Card,
+  CardContent,
+  CardHeader,
+  Chip,
+  CircularProgress,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  Grid,
+  IconButton,
+  Tooltip,
+  Typography,
+} from "@material-ui/core";
+import Link from "@material-ui/core/Link";
+import Paper from "@material-ui/core/Paper";
+import CheckIcon from "@material-ui/icons/Check";
+import ClearIcon from "@material-ui/icons/Clear";
+import EditIcon from "@material-ui/icons/Edit";
+import PersonAddDisabledIcon from "@material-ui/icons/PersonAddDisabled";
+import PersonAddIcon from "@material-ui/icons/PersonAdd";
```

Comment:
Group these imports

Reply:
Some of the icons aren't default exported so I could only group some of them, but I'll group what I can